macOS/LocalPackages/SwiftUIExtensions/Sources/SwiftUIExtensions/SlidingPickerView.swift

```diff
@@ -157,13 +158,13 @@ private extension SlidingPickerView {
         index < allValues.count - 1
     }
 
-    func refreshHighlight() {
+    func refreshHighlight(allowsAnimations: Bool = true) {
         guard let buttonFrame = buttonFrames[selectedIndex] else {
             return
         }
 
         // Note: Avoid animating from Zero Size -> Actual Size
-        animationsEnabled = highlightSize != .zero && settings.animationsEnabled
+        animationsEnabled = highlightSize != .zero && settings.animationsEnabled && allowsAnimations
```

Comment:
This change was required to prevent the Slider Highlight (the "Outer Ring") from animating, when the user resizes the Window